### DISCUSSION TITLE
Methods to watch your IPs for blacklisting?

### DISCUSSION THREAD

**Discussion by NorseGaud** [docker-mailserver/docker-mailserver#2435] [General]
Hey y'all, I've been using the free blacklist scanner from MXToolBox for a while, but wondered what other options there are.

**Comment by casperklein** [comment]
If you are looking for an automated way, there is a [nmap](https://nmap.org/) [script](https://nmap.org/nsedoc/scripts/dns-blacklist.html) you could try.
